src/controllers/action-queue/display.ts

```diff
@@ -100,7 +104,14 @@ async function getActionOpDisplayInfo(account: EdgeAccount, actionOp: ActionOp,
       return {
         ...baseDisplayInfo,
         title: sprintf(s.strings.action_queue_display_swap_title, fromCurrencyCode, toCurrencyCode),
-        message: sprintf(s.strings.action_queue_display_swap_message, fromCurrencyCode, toCurrencyCode)
+        message: sprintf(
+          s.strings.action_queue_display_swap_message,
+          fromCurrencyCode,
+          config.appName,
+          toCurrencyCode,
+          s.strings.loan_aave_fragment,
+          borrowEngineWallet.currencyInfo.currencyCode
```

Comment:
You could just change this to `toWallet.currencyInfo.currencyCode` and omit the op code change.

Reply:
I think it's fine to keep this explicit. I can think of a use case come up soon about wanting to swap for depositing, in which toWallet is no longer valid.

Reply:
`swap` action op is AAVE agnostic. Unless we want to rename it to `swap-aave` and implement the generic action op later (kinda like what we did with wyre-sell over fiat-sell).